src/clj/muuntaja/protocols.clj

```diff
@@ -2,7 +2,15 @@
   (:require [clojure.java.io :as io]
             [muuntaja.util :as util])
   (:import (clojure.lang IFn AFn)
-           (java.io ByteArrayOutputStream ByteArrayInputStream InputStreamReader BufferedReader InputStream Writer)))
+           (java.io ByteArrayOutputStream ByteArrayInputStream InputStreamReader BufferedReader InputStream Writer OutputStream)))
+
+(deftype ByteResponse [bytes]
+  IFn
```

Comment:
I see that `StreamableResponse` below is callable/`IFn`, but honestly I don't remember why. When are you supposed to call these?

Reply:
With an OutputStream, like `System/out`?Could be huge lazy dataset, but I guess `io/input-stream` already handles that. Maybe we should remove those `IFn`s?